.eslint-dictionary.json

```diff
@@ -85,6 +85,8 @@
   "columnify",
   "commitlint",
   "commonjs",
+  "configxc",
```

Comment:
nit: should this 2 be here?

Reply:
This needs to stay because it's referencing something specific to xcode configuration.